README.md

```diff
@@ -1,2 +1,27 @@
-# newspack-plugin
-The Newspack plugin. https://newspack.blog/
+# Newspack
+
+Welcome to the Newspack plugin repository on GitHub. Here you can browse the source, look at open issues and keep track of development. We also recommend everyone [follow the Newspack blog](https://newspack.blog/) to stay up to date about everything happening in the project.
+
+Newspack is an open-source publishing platform built on WordPress for small to medium sized news organizations. It is an “opinionated” platform that stakes out clear, best-practice positions on technology, design, and business practice for news publishers.
```

Comment:
Should we use "plugin" here rather than "platform" as the latter is a much broader thing than just what's in this repo?

Reply:
The plugin and theme are part of the Newspack "platform", and I'm trying to give a 2-sentence overview of the whole thing. I think a couple sentences about the plugin specifically is a good idea, and should resolve some confusion. I'll make that update.